.circleci/config.yml

```
@@ -81,6 +81,29 @@ commands:
           event: success
           condition: on_success
 
+  run_test_byte_code:
+    description: "Run test byte code on a environment."
+    parameters:
+      os_type:
+        type: enum
+        enum: ["ubuntu", "osx"]
```

This does not really control which OS this runs on, only file naming, so I think it would be better to just call this `name` or `label`.
```suggestion
      label:
        type: string
```